src/passthrough/utils.rs

```diff
@@ -14,11 +13,13 @@ pub const HEALTH_PORT_LOWER: u16 = 32768;
 pub const HEALTH_PORT_UPPER: u16 = 33767;
 pub const EPHEMERAL_PORT_LOWER: u16 = 33768;
 pub const EPHEMERAL_PORT_UPPER: u16 = 61000;
-
-pub const ETHERNET_HEADER_LEN: usize = 14;
 pub const IPV4_HEADER_LEN: usize = 20;
 pub const TCP_HEADER_LEN: usize = 32;
 
+// leave for reference
```

Comment:
Another option is to use `#[allow(dead_code)]`  and it is quite common in libs

Reply:
Thats good to know, thanks!